src/libs/fileDownload/FileUtils.ts

```diff
@@ -7,11 +7,12 @@ import type {ReadFileAsync, SplitExtensionFromFileName} from './types';
 
 /**
  * Show alert on successful attachment download
+ * @param successMessage
  */
-function showSuccessAlert() {
+function showSuccessAlert(successMessage?: string) {
     Alert.alert(
         Localize.translateLocal('fileDownload.success.title'),
-        Localize.translateLocal('fileDownload.success.message'),
+        successMessage ?? Localize.translateLocal('fileDownload.success.message'),
```

Comment:
An empty `successMessage` would still be displayed here. This caused an issue #43809